source/index.html.erb

```diff
@@ -11,18 +11,24 @@ title: Ralph's Launch Cards
   </ul>
 </details>
 
-<div class="card">
+<div id="card" class="card">
+
   <header id="card-topic-header" class="card__category">
-    Management
+    Ralph's Launch Cards
   </header>
+
   <p class="card__prompt" id="card-prompt">
-    How's work?
+    Click launch to launch a prompt!
   </p>
+
   <button class="card__refresh-button" id="refresh-button">
     Launch
   </button>
-</div>
 
-<footer>
   <p class="credits">created by ricky and thoughtbot</p>
-</footer>
+
+</div>
+
+<div class="card--layers">
```

Comment:
`--layers` makes me think this is a modifier of `card`. Modifiers are usually chained, so this could be`<div class="card card--layers">`

That way we can inherit most of the default styling from `card`, and then just have the difference/overrides in `card--layers`